cirq/optimizers/eject_z_test.py

```diff
@@ -261,3 +262,37 @@ def with_qubits(self, *new_qubits):
             cirq.Moment([cirq.Z(q)]),
             cirq.Moment([u]),
         ]))
+
+
+def test_swap():
+    a, b = cirq.LineQubit.range(2)
+    original = cirq.Circuit.from_ops([cirq.Rz(.123).on(a), cirq.SWAP(a, b)])
+    optimized = original.copy()
+
+    cirq.EjectZ().optimize_circuit(optimized)
+    cirq.DropEmptyMoments().optimize_circuit(optimized)
+
+    assert optimized[0].operations == (cirq.SWAP(a, b),)
+    assert optimized[1].operations == (cirq.Z(b)**(.123 / np.pi),)
```

Comment:
Why'd you switch from `Rz` to the exponentiated notation?

Reply:
No good reason. It should work though

Reply:
I think for readability you should keep it consistent.